File: 백준/Bronze/4758. Filling Out the Team/Filling Out the Team.py
while 1:
    a,b,c=input().split()
    a,b,c=float(a),int(b),int(c)
    if a==0 and b==0 and c==0: break
    tmp=[]
    chk=0
    if a<=4.5 and b>= 150 and c>=200: 
        tmp.append("Wide Receiver")
        chk=1
    if a<=6.0 and b>= 300 and c>=500: 
        tmp.append("Lineman") 
        chk=1
    if a<=5.0 and b>= 200 and c>=300: 
        tmp.append("Quarterback")
        chk=1
    if chk==0: 
        tmp.append("No positions")
    print(*tmp)

File: 백준/Bronze/4758. Filling Out the Team/Filling Out the Team.js
const readline = require('readline');
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

function Myron(spd, wet, stt) {
  const lst = [];

  if (spd <= 4.5 && wet >= 150 && stt >= 200) {
    lst.push('Wide Receiver');
  }
  if (spd <= 6.0 && wet >= 300 && stt >= 500) {
    lst.push('Lineman');
  }
  if (spd <= 5.0 && wet >= 200 && stt >= 300) {
    lst.push('Quarterback');
  }

  return lst.length > 0 ? lst.join(' ') : 'No positions';
}

rl.on('line', (input) => {
  const [spd, wet, stt] = input.split(' ').map(parseFloat);
  if (spd === 0 && wet === 0 && stt === 0) {
    rl.close();
    return;
  }

  const result = Myron(spd, wet, stt);
  console.log(result);
});

rl.on('close', () => {
  process.exit(0);
});